modules/ensemble/lib/widget/widget_registry.dart

```diff
@@ -135,6 +136,7 @@ class WidgetRegistry {
         PopupMenu.type: () => PopupMenu(),
         EnsembleCalendar.type: () => EnsembleCalendar(),
         Countdown.type: () => Countdown(),
+        Collapsible.type: () => Collapsible(),
```

Comment:
why not call it "Accordion"?

Reply:
We already have this widget in **Ensemble React** as `Collapsible` widget with a flag name `isAccordion`,  then @kmahmood74 mentioned to keep it consistent with **Ensemble React**.